tools/harness-simulation/README.md

```diff
@@ -34,7 +40,7 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 
    ```bash
    $ cd tools/harness-simulation/posix
-   $ python harness_dev_discovery.py \
+   $ python startup_and_discovery.py \
```

Comment:
```suggestion
   $ python launch-testbed.py \
```

Maybe it's a better name?
Thoughts? @superwhd @jcdong98

Reply:
I agree.

Reply:
Agree.